libs/dh/market-participant/actors/feature-actors/src/lib/drawer/balance-responsible-relation-tab/dh-balance-responsible-relation-filter.component.ts

```diff
@@ -139,50 +155,32 @@ export class DhBalanceResponsibleRelationFilterComponent implements OnInit {
     'asc'
   );
 
-  filterForm!: FormGroup<Filters>;
+  filtersForm = new FormGroup<Filters>({
+    status: dhMakeFormControl(),
+    energySupplierWithNameId: dhMakeFormControl(),
+    balanceResponsibleWithNameId: dhMakeFormControl(),
+    gridAreaCode: dhMakeFormControl(),
+    search: dhMakeFormControl(),
+  });
 
   constructor() {
     effect(() => {
-      const {
-        status,
-        energySupplierWithNameId,
-        balanceResponsibleWithNameId,
-        gridAreaCode: gridAreaId,
-        search,
-      } = this.inital();
-
-      this.filterForm.patchValue(
-        {
-          status: status ?? null,
-          energySupplierWithNameId: energySupplierWithNameId ?? null,
-          balanceResponsibleWithNameId: balanceResponsibleWithNameId ?? null,
-          gridAreaCode: gridAreaId ?? null,
-          search: search ?? null,
-        },
-        { emitEvent: false }
-      );
+      // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Alternative suggestion: 

toObservable(this.actor)
      .pipe(takeUntilDestroyed())
      .subscribe(() => this.filtersForm.reset());

Reply:
Love it! Waaay better!